Octokit.Tests.Integration/Clients/DeploymentsClientTests.cs

```diff
@@ -17,33 +19,38 @@ public DeploymentsClientTests()
 
         _deploymentsClient = github.Repository.Deployment;
         _context = github.CreateRepositoryContext("public-repo").Result;
+        _commits = new List<Commit>();
 
-        var blob = new NewBlob
+        for (int i = 0; i < 6; i++)
```

Comment:
Given not all of the tests need this range of commits, do you think it'd be useful to introduce some new code here for the more complex deployment scenarios?